### PR TITLE
fix(navigation): make navigations/tabs view support BC tab params

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: views/default/navigation/tabs.php [hunk 1]
```diff
@@ -19,6 +19,31 @@
 		continue;
 	}
 	
+	// BC tab params
+	if (!isset($tab['text']) && isset($tab['title'])) {
+		$tab['text'] = $tab['title'];
```
Comment (hypeJunction) [inline views/default/navigation/tabs.php]: Maybe put deprecation notices on all of these?

Comment (jdalsem) [reply views/default/navigation/tabs.php]: done

File: views/default/navigation/tabs.php [hunk 2]
```diff
@@ -19,6 +19,38 @@
...
+	}
+
+	if (!isset($tab['href']) && isset($tab['url'])) {
+		elgg_deprecated_notice("Passing 'url' to the navigation/tabs tab is deprecated. Use 'href' instead.", '3.0');
+		$tab['href'] = $tab['url'];
+		unset($tab['url']);
+	}
+
+	if (isset($tab['id']) && isset($tab['link_id'])) {
+		elgg_deprecated_notice('The navigation/tabs tab does not support passing 2 ids. Only id is used to set the id of the menu item', '3.0');
```
Comment (jeabakker) [inline views/default/navigation/tabs.php]:
```suggestion
		elgg_deprecated_notice('The navigation/tabs tab does not support passing 2 ids. Only 'id' is used to set the id of the menu item', '3.0');
```

File: views/default/navigation/tabs.php [hunk 3]
```diff
@@ -19,6 +19,38 @@
...
+	}
+
+	if (!isset($tab['href']) && isset($tab['url'])) {
+		elgg_deprecated_notice("Passing 'url' to the navigation/tabs tab is deprecated. Use 'href' instead.", '3.0');
+		$tab['href'] = $tab['url'];
+		unset($tab['url']);
+	}
+
+	if (isset($tab['id']) && isset($tab['link_id'])) {
+span class="pl-s1"> 		elgg_deprecated_notice('The navigation/tabs tab does not support passing 2 ids. Only 'id' is used to set the id of the menu item', '3.0');
```
Comment (jdalsem) [inline views/default/navigation/tabs.php]:
```suggestion
 		elgg_deprecated_notice("The navigation/tabs tab does not support passing 2 ids. Only 'id' is used to set the id of the menu item", '3.0');
```

**Review by jeabakker — APPROVED** [review]

**Pull request merged**